src/ripple/core/tests/Config.test.cpp

```diff
@@ -138,28 +128,26 @@ backend=sqlite
 class ConfigGuard
 {
 private:
+    bool rmSubDir_{false};
+
+protected:
     using path = boost::filesystem::path;
     path subDir_;
-    path configFile_;
-    path dataDir_;
 
-    bool rmSubDir_{false};
-    bool rmDataDir_{false};
-
-    Config config_;
+    auto rmDir (path const& toRm)
+    {
+        if (is_directory (toRm) && is_empty (toRm))
+            remove (toRm);
+        else
+            std::cerr << "Expected " << toRm.string ()
+                      << " to be an empty existing directory.\n";
```

Comment:
I didn't realize on the first pass that `rmDir` is not called directly from the test suite. The way I would implement this is to add a `beast::unit_test::suite& test` member to `ConfigGuard`, initialize it from the constructors, and pass `*this` when you're constructing inside the unit test functions. Then use `test.log << ...` here.

On second thought, show don't tell. Here's a commit: https://github.com/ximinez/rippled/commit/523472f0b980d572f0c95c60c55d4d19787b0043